imjpfeliciano/coto-admin-dashboard
# Feat: Add test config

## What does this PR do?

- This PR includes base configuration to run unit and e2e testing using jest and playwright respectively


## Why is it important?

This will help us to check nothing breaks when introducing new changes to the codebase

## How to test it locally

* Check this branch out
* Install dependencies (`yarn`)
* execute unit tests: `yarn test:ci`
* execute e2e tests: `yarn test:e2e`

## Screenshots (if appropriate)

NA

## Related issues

#2
